continuous-integration/requirements.Rmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Requirements for continuous integration"
+maintainer: "Thierry Onkelinx"
+output: html_document
+---
+
+# Must have
+
+- start automatically when pushing a git repository on Github or Bitbucket
+- start automatically when creating a pull request on Github or Bitbucket
+- able to use a Docker image to setup the environment in which CI runs
```

Comment:
I'm wondering:
* Is the usage of Docker properly supported on an INBO organisation level?
* Will we run into pricing requirements for Docker at a certain level?
* Is this only useful for development or how should I see this? How is it positioned? When would someone need/use a Docker image? 
* How does this relate to the work of researchers? How do they have to use this?